packages/core/utils/src/dml/integration-tests/__tests__/many-to-many.spec.ts

```diff
@@ -186,4 +186,40 @@ describe("manyToMany - manyToMany", () => {
       ]),
     })
   })
+
+  it(`should fail to load the dml's if both side of the relation are missing the mappedBy options`, () => {
+    const team = model.define("team", {
+      id: model.id().primaryKey(),
+      name: model.text(),
+      users: model.manyToMany(() => user, {
```

Comment:
q: the `user` model doesn't have a `teams.manyToMany`. Is the inverse optional?

Reply:
It has the squad on the user model which is the one that refers to the team, now I haven't named it team but it is the relation that is important. So you can add mappedby on the squads to users or on team to squads. does that make sense?

Reply:
Ah, I see. I missed that. So then follow up question, if the inverse is not present, does that throw an error? i.e. if the user.squads relationship did not exist

Reply:
No you can't but it is true that we don't check that, mikro orm wont complain if you do that, but you wont be able to have a real many to many and the result of querying will be missing the other side relation even though you can specify it as part of the populate, let me add that

Reply:
I ve updated the check, you can have another look

Reply:
Nice, looks good man! 🔥

Reply:
thanks to you for mentionning the missing part 😄